### PR TITLE
refactor: remove obsolete code

### PR DESCRIPTION
**Pull request by kade-ddnkv**

### CODE REVIEW

**Comment by kade-ddnkv** [comment]
0-Scenario is working

**Comment by kade-ddnkv** [comment]
I would like to place models into different folder (cmd/api/rest/models) to retain the beauty of good decomposition, but I don't like the way imports are going to be.

**Comment by kade-ddnkv** [comment]
0-Scenario is working

File: cmd/ebpf/objects.go [hunk 1]
```diff
@@ -172,17 +140,7 @@ func ResizeEbpfMap(eMap **ebpf.Map, eProg *ebpf.Program, newSize uint32) error {
...
-	}
-	// FAR
-	if err := ResizeEbpfMap(&bpfObjects.FarMap, bpfObjects.UpfFarProgramFunc, farMapSize); err != nil {
-		log.Info().Msgf("Failed to resize far map: %s", err)
-		return err
-	}
+func (bpfObjects *BpfObjects) ResizeAllMaps(pdrMapSize uint32) error {
 	// PDR
 	if err := ResizeEbpfMap(&bpfObjects.PdrMapDownlinkIp4, bpfObjects.UpfIpEntrypointFunc, pdrMapSize); err != nil {
 		log.Info().Msgf("Failed to resize qer map: %s", err)
```
Comment (pirog-spb) [inline cmd/ebpf/objects.go:146]: Something wrong with comment

File: cmd/ebpf/objects.go [hunk 2]
```diff
@@ -172,17 +140,7 @@ func ResizeEbpfMap(eMap **ebpf.Map, eProg *ebpf.Program, newSize uint32) error {
 	return nil
 }
 
-func (bpfObjects *BpfObjects) ResizeAllMaps(qerMapSize uint32, farMapSize uint32, pdrMapSize uint32) error {
-	// QEQ
-	if err := ResizeEbpfMap(&bpfObjects.QerMap, bpfObjects.UpfQerProgramFunc, qerMapSize); err != nil {
```
Comment (pirog-spb) [inline cmd/ebpf/objects.go:177]: I suppose resizing of far and qer should stay here but need to be fixed

File: Dockerfile [hunk 3]
```diff
@@ -25,7 +25,6 @@ LABEL org.opencontainers.image.source="https://github.com/edgecomllc/eupf"
 
 COPY --from=builder /app/bin/ /app/bin/
 COPY --from=builder /app/cmd/docs/swagger.* /app/
-COPY --from=builder /app/cmd/ebpf/zeroentrypoint_bpf.o /app/
```
Comment (pirog-spb) [inline Dockerfile:28]: `zeroentrypoint_bpf` is used sometimes

Comment (kade-ddnkv) [reply Dockerfile:28]: +

File: cmd/ebpf/objects.go (hunk 1 above)
Comment (kade-ddnkv) [reply cmd/ebpf/objects.go]: \+

File: cmd/ebpf/objects.go (hunk 2 above)
Comment (kade-ddnkv) [reply cmd/ebpf/objects.go:177]: Should something be done with FarIdTracker and QerIdTracker too? Their map size is being set only on initialization

Comment (pirog-spb) [reply cmd/ebpf/objects.go:177]: Resizing works correctly on initialization only

**Pull request merged**